.changeset/plenty-crews-wink.md

```diff
@@ -0,0 +1,6 @@
+---
+"@khanacademy/perseus": major
+"@khanacademy/perseus-editor": major
+---
+
+Remove mock widgets and their editors
```

Comment:
Maybe refer to "example" instead of "mock" as the mocks were really only for unit tests, right? 

```suggestion
Remove example widgets and their editors
```